### DISCUSSION TITLE
Right now users are limited to 10 installs but I was thinking about making that a number you can customize. What would you want to set it to?

### DISCUSSION THREAD

**Discussion by vinay-sd17** [Glench/ExtPay#154] [Ideas]
Any possibility to make this limit number customizable Glen? Getting some frequent issues with Too Many Login attempts error by many users of mine. (Dev & Prod)

------------------------------------------------------------------------------------

              Right now users are limited to 10 installs but I was thinking about making that a number you can customize. What would you want to set it to?

I don't have any plans at the moment to have ExtensionPay support a "seat"-based payment system but that may change if more people ask for it. Please comment here or email me if that's something you're needing and explain your use case!

_Originally posted by @Glench in https://github.com/Glench/ExtPay/issues/18#issuecomment-978215372_

**Comment by Glench** [comment]
Yeah, this part of the system needs to be changed for sure. And to be clear, this has nothing to do with the quote of mine you posted about seat-based pricing. That is a completely separate issue.

**Reply by vinay-sd17** [reply]
Hi, Seat-based pricing is not needed. At least if we can have this option it will be nice.

         - Right now users are limited to 10 installs but I was thinking about making that a number you can customize. What would you want to set it to?

**Comment by JPaulHendricks** [comment]
i have not yet implemented ExtensionPay yet, but plan to shortly. Is the above "limited to 10 installs" a default? 

This sounds like a subscription permits multiple installs. I assumed that is NOT the case; there can be multiple installs but only one login at a time. Is that the case? IOW one active login would get booted out /de-authorized if another login/auth attempt was done. Is that correct?

**Reply by Glench** [reply]
I answered this in a reply to you on a different thread, but I think it's more useful for you to think of an "install" as one person trying to rightfully use the paid functionality of your extension on, for example, multiple chrome profiles or different devices they own, or if they uninstall then reinstall the extension. If this was not how it worked, users would be much more frustrated and likely to complain about your extension.

That said, we're working to find a good middle ground. Stay tuned.

Here's my reply in the other thread:

> This is not how it works currently but we'll change it to be a little more like you described. The problem is that "disable the earlier auth/usage" is actually a terrible user experience. If one user wants to use the extension on multiple chrome profiles, for example, then if they're limited to 1 active use they have to sign in every time they switch chrome profiles. Most users that pay expect to be able to use their extension's paid functionality however they want. 10 installs is an upper limit meant to limit abuse while also minimizing user frustration (which is much more common than abuse, by the way).

**Comment by scottblair** [comment]
It would be great to have this as a setting we can adjust, but for my purposes... having it limited to 5 would be ideal.
